2009-03-13  Adrien DELLE CAVE <decryptus@proformatique.com>

	pf_macro_loop_verbose.patch
	$Revision$
	$Date$

	Added Macro() arguments information in log if there is a possible infinite loop.

Index: asterisk-1.4.24/apps/app_macro.c
===================================================================
--- asterisk-1.4.24.orig/apps/app_macro.c	2009-05-27 12:56:40.000000000 +0200
+++ asterisk-1.4.24/apps/app_macro.c	2009-05-27 12:58:23.000000000 +0200
@@ -228,7 +228,7 @@
 		sscanf(inhangupc, "%d", &inhangup);
 
 	if (depth >= maxdepth) {
-		ast_log(LOG_ERROR, "Macro():  possible infinite loop detected.  Returning early.\n");
+		ast_log(LOG_ERROR, "Macro(%s): possible infinite loop detected. Returning early.\n", (char *)data);
 		ast_module_user_remove(u);
 		return 0;
 	}
